### PR TITLE
jr 0.3.3 (new formula)

### PR DESCRIPTION
**Pull request by ram-pi**
Bugfixing and support for write_env_script

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/j/jr.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  license "MIT"
+  head "https://github.com/ugol/jr.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "all"
+    libexec.install Dir["build/*"]
+    prefix.install "config/jrconfig.json"
+    prefix.install "templates"
```
Comment (SMillerDev) [inline Formula/j/jr.rb:15]: Can these not go into the usual `etc` directories for configuration and `share` for application files?

Comment (ram-pi) [reply Formula/j/jr.rb:15]: I think I can put both under `etc`.

Comment (SMillerDev) [reply Formula/j/jr.rb:15]: But templates aren't config so they should go to `share`

Comment (ram-pi) [reply Formula/j/jr.rb]: The problem is that I can set just one `JR_HOME` and the tool is supposed to find under its home the needed files. I cannot split in two folders as for now.

File: Formula/j/jr.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  license "MIT"
+  head "https://github.com/ugol/jr.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "all"
+    libexec.install Dir["build/*"]
+    # prefix.install "config/jrconfig.json"
+    # prefix.install "templates"
```
Comment (chenrui333) [inline Formula/j/jr.rb:15]:
```suggestion
```

**Comment by chenrui333** [comment]
Also please squash the commits into one with commit msg as `jr 0.3.3 (new formula)`. Thanks!

**Pull request closed without merging**